src/SDKs/CognitiveServices/dataPlane/Vision/FormRecognizer/FormRecognizer/Microsoft.Azure.CognitiveServices.Vision.FormRecognizer.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Vision.FormRecognizer</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services FormRecognizer APIs.</Description>
+    <Version>2.4.0-preview</Version>
```

Comment:
Versioning must start from `0.8.0-preview`

Reply:
done. Fixed to v1.0.0.0